nautobot/dcim/tests/test_models.py

```diff
@@ -1423,6 +1429,61 @@ def test_create_cable_with_missing_status_connected(self):
 
         self.assertTrue(Cable.objects.filter(id=cable.pk).exists())
 
+    def test_deleting_device_with_multiple_types_of_connected_interfaces_successful(self):
+        Cable.objects.all().delete()
+        interface1 = Interface.objects.create(device=self.device1, name="ethernet1")
+        interface2 = Interface.objects.create(device=self.device2, name="ethernet2")
+        interface3 = Interface.objects.create(device=self.device1, name="ethernet3")
+        interface4 = Interface.objects.create(device=self.device2, name="ethernet4")
```

Comment:
Missing a `status` on these Interfaces. Can/should we use the `self.interfaceX` interfaces already created in `setUpTestData` instead of creating new ones?